frontend/src/views/instructor/routes/index.tsx

```diff
@@ -1,27 +1,26 @@
 import React from "react";
 import { Redirect, Route, Switch } from "react-router-dom";
-import { InstructorPositionsView } from "../positions";
 import { InstructorAssignmentsView } from "../assignments";
 import { Landing as InstructorLanding } from "../landing";
+import { InstructorDDAHsView } from "../ddahs";
 
 export function InstructorRoutes() {
     return (
         <Switch>
             <Route exact path="/">
-                <Redirect to="/tapp" />
-            </Route>
-            <Route exact path="/tapp">
                 <InstructorLanding />
             </Route>
-            <Route exact path="/tapp/positions">
-                <InstructorPositionsView />
-            </Route>
-            <Route exact path="/tapp/assignments">
+            <Route
+                path="/:positionId"
+                render={({match}) => {
+                    const positionId = match.params.positionId;
+                    return <Redirect to={`/${positionId}/assignments`} />;
+                }}
+            />
+            <Route path="/:positionId/assignments">
                 <InstructorAssignmentsView />
             </Route>
```

Comment:
This `InstructorAssignmentsView` component seems to not render, and I have no idea why, was thinking it's redux's `shouldComponentUpdate()`, but wrapping in `withRouter` did not help. Any thoughts @siefkenj ?